src/ReactDOMServer.re

```diff
@@ -4,3 +4,24 @@ external renderToString: React.element => string = "renderToString";
 [@bs.module "react-dom/server"]
 external renderToStaticMarkup: React.element => string =
   "renderToStaticMarkup";
+
+type error = {.};
+
+[@bs.module "react-dom/server"]
+external renderToPipeableStream:
```

Comment:
this seems wrong. According to the TypeScript type definitions it should be a function of 2 arguments: a ReactNode and an options object:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f1c7be0bf694f55a4cb8b34bd1134e3d23cf21e7/types/react-dom/server.d.ts#L27

Reply:
Totally, I had copied my implementation from server-reason-react and forgot to bind it properly